src/main/java/org/cactoos/iterator/Repeated.java

```diff
@@ -62,16 +62,16 @@ public Repeated(final int max, final T element) {
      * @param max How many times to repeat
      * @param scalar Scalar to repeat
      */
-    public Repeated(final int max, final Scalar<T> scalar) {
-        this(max, new Unchecked<T>(scalar));
+    public Repeated(final int max, final Scalar<? extends T> scalar) {
+        this(max, new Unchecked<>(scalar));
     }
 
     /**
      * Ctor.
      * @param max How many times to repeat
      * @param scalar Scalar to repeat
      */
-    private Repeated(final int max, final Unchecked<T> scalar) {
+    private Repeated(final int max, final Unchecked<? extends T> scalar) {
```

Comment:
@DmitryBarskov I suggest removing this ctor, and doing `this.elm = new Unchecked<>(...)` in ctor itself